RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate CannedResponsesListView to Typescript

## Proposed changes

Migrate CannedResponsesListView and CannedResponseDetail to TS

## Issue(s)	


## How to test or reproduce

- You need to be a Livechat agent
- Then log in, take a channel, click on the title header and click on Canned Responses button
- You should be moved to Canned Response List View, and when click in any Canned response, you will be redirect to Canned Response Detail
- Both views were migrated.

## Screenshots

## Types of changes




## Checklist



## Further comments
